controller/src/main/java/cern/jarrace/controller/Application.java

```diff
@@ -0,0 +1,25 @@
+package cern.jarrace.controller;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ApplicationContext;
+
+/**
+ * Main entry point for the controller, instanciates an embedded Jetty container server, loads the spring
+ * {@link org.springframework.web.servlet.DispatcherServlet} and configures all the
+ * {@link org.springframework.web.bind.annotation.RestController}s
+ * @author timartin
+ */
+
+@SpringBootApplication
+public class Application {
```

Comment:
I like the name Application cause its the main entry point for the application, but it can easily be changed for a better one.
Any sugestion?